Documentation/Programmatically.md

```diff
@@ -188,4 +188,15 @@ Workbook types specify which workbook gallery type the new workbook instance wil
 | `usage` | The _More_ gallery under _Usage_ in Application Insights |
 
 ### Limitations
```

Comment:
while you're touching this file, can you remove this limitations section entirely?  as of the legacy workbook merges, this limitation is no longer true